docker-compose.yml

```diff
@@ -207,6 +212,11 @@ services:
             - backend.corporanet.local
   oidc:
     image: soluto/oidc-server-mock:0.3.0
+    # For Mac M1, you may need to build this image from source.
```

Comment:
`may need to`? 🤔

Reply:
I figured this is a YMMV sort of thing. But if others can confirm that the pre-built oidc-server-mock image is failing as well on their M1s, I'll remove the indecisive language.